mst/postgresql-read-replica.md

```diff
@@ -38,3 +39,37 @@ When the read-only replica is created it is listed as a service in your
 project. The `Overview` tab of the replica also lists the name of the primary
 service for the replica. To promote a read-only replica as a master database,
 click the `Promote to master` button.
+
+<procedure>
+
+## Using read-only replica for the service on MST
+
+1.  In the `Overview` page of the read-only replica for the service on MST, copy
+    the `Service URI`.
+
+1.  Connect to the read-only service using psql client:
+
+    ```sql
+       psql <SERVICE_URI>
```

Comment:
Line the command up with the three backticks

Reply:
done